app/views/templates/search/search_object.rb

```diff
@@ -297,25 +383,52 @@ def date_data
 
       
       def provenance_data
-        # origin
 
-        originsPublisher     = collect_values(['proxies.dcPublisher'])
-        originsProvider      = collect_values(['aggregations.edmProvider'])
-        originsDataProvider  = collect_values(['aggregations.edmDataProvider'])
-        originsCountry       = collect_values(['europeanaAggregation.edmCountry'])
-        originsOther         = collect_values(['proxies.dctermsProvenance', 'proxies.dcSource', 'proxies.dctermsReferences', 'proxies.dcIdentifier'])
-          
-        origins = [].push(*originsPublisher).push(*originsProvider).push(*originsDataProvider).push(*originsCountry).push(*originsOther)
+        require 'time'
+
+        origins_publisher     = collect_values(['proxies.dcPublisher'])
+        origins_provider      = collect_values(['aggregations.edmProvider'])
+        origins_provenance    = collect_values(['proxies.dctermsProvenance'])
+        origins_data_provider = collect_values(['aggregations.edmDataProvider'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.